docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,131 @@
 
 # Quickstart Guide
 
+## Dependencies
+
+Please ensure you have the following dependencies installed:
+
+* [gen3-client](../requirements/gen3-client.md)
+* [gen3-tracker](../requirements/gen3-tracker.md)
+
+## Usage
+
+`g3t  [OPTIONS] COMMAND [ARGS]...`
+
+The following options and environmental variables are synonymous, you may set them as environmental variables or pass them as parameters to the command line.
+
+| option       | environment     | comment             | example       |
+|--------------|-----------------| ------------------- |---------------|
+| --project_id | G3T_PROJECT_ID  | authorization       | aced-test     |
+| --profile    | G3T_PROFILE     | gen3-client profile | production    |
+| --format     | G3T_FORMAT      | Output format.      | default: yaml |
+
+Alternatively, you can set the environmental variables using the `EXPORT` function e.g.:
+
+```sh
+export G3T_PROJECT_ID=aced-myproject
+```
+
+
 ## 1. Upload Data (To an Existing Project on ACED IDP)
-This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarily to `git` commands
+This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarly to `git` commands
+
+### Initialize a new project:
 
-Initialize a new project
 ```
 g3t init aced-demo
 ```
 
-Clone an existing project
-```
-g3t clone aced-demo
-```
+* Similar to `git init`, this command initializes a new project in the current directory
+* The id is `aced-demo`; the program name is `aced`, the project name is `demo`
+* `Program name` is predefined by the institution, the `g3t ping` command will show the available programs
+* `Project name` must be unique within the server, and must be alphanumeric and contain no spaces
+
+
+### Add file(s):
 
-Add file(s)
 ```
 g3t add folder/file.tsv
 g3t add folder/file2.tsv
 ```
 
-Create meta data for your files. This `g3t` command will take care of creating this for you
+* This command will record an entry for each file in the `MANIFEST` directory.
+* Optionally you can tag each file with patient, specimen or other identifiers. See `g3t add --help` for more information.
```

Comment:
Shall we link to [/workflows/upload/](https://aced-idp.github.io/workflows/upload/) so they can stay in the docs as opposed to needing to go into the CLI?

Reply:
done